packages/zapp/console/src/components/Literals/Scalar/StructuredDatasetValue.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  SimpleType,
+  StructuredDataset,
+  StructuredDatasetColumn,
+  StructuredDatasetColumnType,
+} from 'models/Common/types';
+import * as React from 'react';
+import { PrintList } from '../PrintList';
+import { PrintValue } from '../PrintValue';
+import { useLiteralStyles } from '../styles';
+import { ValueLabel } from '../ValueLabel';
+
+function simpleColumnTypeToString(type: SimpleType) {
+  switch (type) {
+    case SimpleType.NONE:
+      return 'none';
+    case SimpleType.INTEGER:
+      return 'integer';
+    case SimpleType.FLOAT:
+      return 'float';
+    case SimpleType.STRING:
+      return 'string';
+    case SimpleType.DATETIME:
+      return 'datetime';
+    case SimpleType.DURATION:
+      return 'duration';
+    case SimpleType.BINARY:
+      return 'binary';
+    case SimpleType.ERROR:
+      return 'error';
+    case SimpleType.STRUCT:
+      return 'struct';
+    default:
+      return 'unknown';
+  }
+}
+
+function columnTypeToString(literalType?: any, level = 0) {
+  if (level === 2) {
+    return literalType?.type;
+  }
+
+  switch (literalType?.type) {
+    case 'collectionType': {
+      return `collection of ${columnTypeToString(literalType.collectionType, ++level)}`;
+    }
+    case 'simple': {
+      return simpleColumnTypeToString(literalType.simple);
```

Comment:
@anrusina what do you think of this vs 
```suggestion
      return (simpleTypeToInputType[literalType.simple] as string).toLocaleLowerCase()
```

Reply:
I like when it lowercased - it looks nice. From pattern perspective first one looks better (mainly because i'm not a big fan of `as` casting :)